src/dev/ci_setup/git_setup.sh

```diff
@@ -99,3 +100,4 @@ function checkout_sibling {
 }
 
 checkout_sibling "elasticsearch" "${PARENT_DIR}/elasticsearch" "USE_EXISTING_ES"
+checkout_sibling "kibana-offline-mirror" "${KIBANA_EXTRA_DIR}/kibana-offline-mirror" "USE_EXISTING_KIBANA_OFFLINE_MIRROR"
```

Comment:
Curious, why put it in the `kibana-extras` directory and not a Kibana sibling, like the elasticsearch checkout?

Reply:
no reason other than our previous convention.  given the kibana prefix i think it makes sense to move it up one